nh_eobs_mental_health/models/nh_clinical_patient_observation_ews.py

```diff
@@ -29,32 +29,8 @@ class NHClinicalPatientObservationEWS(orm.Model):
 
     def complete(self, cr, uid, activity_id, context=None):
         """
-        It determines which acuity case the current observation is in
-        with the stored data and responds to the different policy
-        triggers accordingly defined on the ``_POLICY`` dictionary::
-
-            {'ranges': [0, 4, 6], 'case': '0123', --> Used with bisect to
-            determine the acuity case based on the score.
-            'frequencies': [720, 240, 60, 30], --> frequency of recurrency
-            of the NEWS observation, based on the case.
-            'notifications': [...],
-               Information sent to the trigger_notifications method,
-               based on case.
-            'risk': ['None', 'Low', 'Medium', 'High']} --> Clinical risk
-            of the patient, based on case.
-
-        All the case based lists work in a simple way:
-        list[case] --> value used
-
-        After the policy triggers take place the activity is `completed`
-        and a new NEWS activity is created. Then the case based
-        `frequency` is applied, effectively scheduling it.
-
-        In the case of having a `partial` observation we won't have a new
-        frequency so the new activity is scheduled to the same time the
-        one just `completed` was, as the need for a complete observation
-        is still there.
-
+        Mental health override that ensures Clinical Review tasks are set up
+        for refusing patients at the appropriate time using CRON jobs.
         :returns: ``True``
```

Comment:
Blank line between the description and params list. Otherwise won't render correctly

Reply:
I've been doing this for ages even though I don't like doing it because you said that the Odoo linting (or something like that) complains if there's a blank line.